DigitalOcean.API/Clients/ISnapshotsClient.cs

```diff
@@ -1,23 +1,14 @@
 ﻿using System.Collections.Generic;
 using System.Threading.Tasks;
+using DigitalOcean.API.Models.Requests;
 using DigitalOcean.API.Models.Responses;
 
 namespace DigitalOcean.API.Clients {
     public interface ISnapshotsClient {
         /// <summary>
         /// To list all of the snapshots available on your account.
         /// </summary>
-        Task<IReadOnlyList<Snapshot>> GetAll();
-
-        /// <summary>
-        /// To retrieve only snapshots based on Droplets.
-        /// </summary>
-        Task<IReadOnlyList<Snapshot>> GetAllDroplet();
-
-        /// <summary>
-        /// To retrieve only snapshots based on volumes.
-        /// </summary>
-        Task<IReadOnlyList<Snapshot>> GetAllVolume();
+        Task<IReadOnlyList<Snapshot>> GetAll(SnapshotType type = SnapshotType.All);
```

Comment:
I only recently created this Client. Found the `IImagesClient` deals with listing the multiple Images types using an enum. Seemed most appropriate to use the same format here.